aman/src/components/OuterLayout.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Background from "../assets/background.jpg";
+import ShipGlobal from "../assets/shipGlobal.png";
+import { Link } from "react-router-dom";
+
+const OuterLayout = ({ children }) => {
+  return (
+    <div
+      className="bg-cover h-screen"
+      style={{ backgroundImage: `url(${Background})` }}
+    >
+      <Link to="#" className="grid">
```

Comment:
check if grid classname is used properly.

Reply:
Yes ! using grid gives me flexibility for future alignment also it automatically streches the bg and aligned it .